dnf5/commands/download/download.cpp

```diff
@@ -80,6 +83,12 @@ void DownloadCommand::set_argument_parser() {
     alldeps->set_const_value("true");
     alldeps->link_value(alldeps_option);
 
+    auto source = parser.add_new_named_arg("source");
+    source->set_long_name("source");
+    source->set_description("Download the src.rpm instead");
+    source->set_const_value("true");
+    source->link_value(source_option);
```

Comment:
Can you please rename the option to `--srpm`?

We are trying to unify  the options in dnf5 and we have decided to use `--srpm` everywhere. Old dnf sometimes used `--source` and sometimes `--srpm` which was confusing.

Sorry about the outdated [issue](https://github.com/rpm-software-management/dnf5/issues/947) I have changed it there to `--srpm` as well.